docs/workflow/building/mono/README.md

```diff
@@ -53,6 +53,10 @@ The build has a number of options that you can learn about using build -?.
 
 See the instructions in [../libraries/webassembly-instructions.md].
 
+### Android
+
+See the instructions in [../../testing/libraries/testing-android.md]
+
 ## Packages
```

Comment:
Can you also add a section for iOS as well? 
doc: https://github.com/dotnet/runtime/blob/master/docs/workflow/testing/libraries/testing-apple.md

Reply:
sure